core/helpers/index.ts

```diff
@@ -219,8 +219,10 @@ export const calcItemsHmac = (items, token) => {
 export function extendStore (moduleName: string | string[], module: any) {
   const merge = function (object: any = {}, source: any) {
     for (let key in source) {
-      if (typeof source[key] === 'object') {
+      if (typeof source[key] === 'object' && !Array.isArray(source[key])) {
```

Comment:
Couldn't this be first checked if it is an Array  so you don't need to do it 2 times?

Reply:
@ResuBaka thank's 👍